src/ledger/LedgerTxn.h

```diff
@@ -578,12 +573,9 @@ class AbstractLedgerTxn : public AbstractLedgerTxnParent
     virtual LedgerTxnHeader loadHeader() = 0;
     virtual LedgerTxnEntry create(InternalLedgerEntry const& entry) = 0;
 
-#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
-    virtual LedgerTxnEntry restore(InternalLedgerEntry const& entry) = 0;
-#endif
-
     virtual void erase(InternalLedgerKey const& key) = 0;
-    virtual LedgerTxnEntry load(InternalLedgerKey const& key) = 0;
+    virtual LedgerTxnEntry load(InternalLedgerKey const& key,
+                                bool loadExpiredEntry = false) = 0;
```

Comment:
I made this an optional parameter in the interest of time. We're going to remove this anyways so it should be fine.